.github/workflows/stale.yml

```diff
@@ -60,4 +62,4 @@ jobs:
             recent activity. Thank you for your contributions.
 
         # Max number of operations per run
-        operations-per-run: 30
+        operations-per-run: 100
```

Comment:
Let's keep 30. We are not in a hurry with stale PRs.

Reply:
As you which. 
For the record, with the current state of the stale action, 2 operations are consumed to check the last modification date of all PRs having the stale label. This means conan-center-index will never have more than 15 stale PRs

Reply:
No problem, we still can re-visit a bunch of them. Thank you about this information. 

About the rate-limit, we have been doing some experiments using Github API (internally), so as less as we use in actions better will be.